docs/manual/download.md

```diff
@@ -1,39 +1,32 @@
 
-To download the library, simply clone the MixedReality-WebRTC repository:
 
-```shell
-git clone https://github.com/microsoft/MixedReality-WebRTC.git
-```
+See [Installation](installation.md) for details on how to use NuGet packages in a Visual Studio project.
 
-> [!IMPORTANT]
-> The C++ library distributed in the NuGet packages v1.0.x has an architecture defect : it requires some internal headers from the Google's WebRTC implementation to be used, which are **not** shipped with those NuGet packages. See bug [#123](https://github.com/microsoft/MixedReality-WebRTC/issues/123) for details. A workaround is to clone the repository _recursively_ to get those headers (`git clone --recurse`).
+### Legacy v1 packages
 
-## Unity library
+In v1 the native packages [`Microsoft.MixedReality.WebRTC.Native.Desktop`](https://www.nuget.org/packages/Microsoft.MixedReality.WebRTC.Native.Desktop) and [`Microsoft.MixedReality.WebRTC.Native.UWP`](https://www.nuget.org/packages/Microsoft.MixedReality.WebRTC.Native.UWP) contain a native library exposing a C++ interface. This caused several issues:
 
-The sources for the Unity library are distributed as a UPM package to be imported into a user project via the Unity Package Manager. Users can checkout the GitHub repository and point UPM to the `package.json` file from [`libs/unity/library`](https://github.com/microsoft/MixedReality-WebRTC/tree/master/libs/unity/library/) (library) and [`libs/unity/samples`](https://github.com/microsoft/MixedReality-WebRTC/tree/master/libs/unity/samples/) (optional demo samples). The library however requires the user to compile from sources the C and C# libraries, which the Unity library uses under the hood. Standalone UPM packages with precompiled binaries are not available at this time. See [Installation](installation.md) for details.
+- Compatibility with various C++ compilers and CRT versions, which is a general problem when working with DLLs on Windows platforms.
+- Architecture defect : some library headers were including some internal headers from the Google's WebRTC implementation, which were **not** shipped with those NuGet packages. See bug [#123](https://github.com/microsoft/MixedReality-WebRTC/issues/123) for details.
 
-> [!WARNING]
-> The Unity scripts are using the API of the branch they are associated with. This means that the Unity scripts on the `master` branch are only working with the `master` branch API, and are therefore not compatible with the NuGet packages, which are only provided for stable releases (`release/xxx` branches).
+To avoid those issues, starting from 2.0.0 the native library exposes a pure C interface. To highlight this change and the fact the packages are native, some new package names are used. **The legacy v1 native packages are now deprecated and unsupported, and should not be used.**

```

Comment:
> **The legacy v1 native packages are now deprecated and unsupported, and should not be used.**

Put this at the top of the section - so it's clear that this section is here only to highlight deprecation and not as an alternative system.